docs/book/src/developer/guide.md

```diff
@@ -147,9 +148,18 @@ To test another iteration, you'll need to follow the steps to build, push, updat
 
 **Additional ClusterAPI KubeCon talks**
 
+* [How Adobe Planned For Scale With Argo CD, Cluster API, And VCluster - October 2022](https://www.youtube.com/watch?v=p8BluR5WT5w)
+* [Bare-Metal Chronicles: Intertwinement Of Tinkerbell, Cluster API And GitOps - October 2022](https://www.youtube.com/watch?v=NCFUUjTw6hA)
+* [Running Isolated VirtualClusters With Kata & Cluster API - October 2022](https://www.youtube.com/watch?v=T6w3YrExorY)
+* [SIG Cluster Lifecycle Intro - October 2022](https://www.youtube.com/watch?v=0Zo0cWYU0fM)
 * [How to Migrate 700 Kubernetes Clusters to Cluster API with Zero Downtime - May 2022](https://www.youtube.com/watch?v=KzYV-fJ_wH0)
 * [Build Your Own Cluster API Provider the Easy Way - May 2022](https://www.youtube.com/watch?v=HSdgmcAAXa8)
 
+**Tutorials**
+
+* - [kubectl Create Cluster: Production-ready Kubernetes with Cluster API 1.0 - October 2022](https://kccncna2022.sched.com/event/1BZDs)
```

Comment:
Formatting `* -` looks slightly off

Reply:
It's correctly formatted if that's what you mean. I created a sublist to make it clear it's video + lab link.

Reply:
That looks a bit strange to me :)

<img width="977" alt="image" src="https://user-images.githubusercontent.com/4662360/203801736-426e38ed-e867-4ebd-bc4f-c609c6026fe9.png">

What about just indenting the `source code` items?

Reply:
(or adding it in `()` to the items with the sched links)

Reply:
No problem. That works for me :)